src/vasp.jl

```diff
@@ -165,4 +165,59 @@ end
 
 readDOSCAR(filename::AbstractString) = open(readDOSCAR, filename)
 
-readDOSCAR() = open(readDOSCAR, "DOSCAR")
\ No newline at end of file
+readDOSCAR() = open(readDOSCAR, "DOSCAR")
+
+function readPROCAR(io::IO)
```

Comment:
Include a docstring for the function definition. Feel free to use previous docstrings (such as for `readWAVECAR()`) as a template and change things around to match `readPROCAR()`.